perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/modules/Vsup.java

```diff
@@ -159,6 +194,31 @@ public Application approveApplication(PerunSession session, Application app) thr
 
 		}
 
+		// create ues RC for future merging
+		List<ApplicationFormItemData> data = new ArrayList<>();
+		try {
+			data = registrar.getApplicationDataById(session, app.getId());
+		} catch (RegistrarException e) {
+			// ignore because application's id is not null
+		}
+		for (ApplicationFormItemData item : data) {
+			if (item.getFormItem() != null &&
+				Objects.equals(AttributesManager.NS_USER_ATTR_DEF + ":birthNumber", item.getFormItem().getPerunDestinationAttribute())) {
+
+				String rc = item.getValue();
+				if (rc != null && !rc.isEmpty()) {
+					ExtSource es = perun.getExtSourcesManager().checkOrCreateExtSource(session, "RC", rc);
```

Comment:
This would create ExtSource with name RC and type as value of birthNumber. At VŠUP type of RC ExtSource is ``INTERNAL``, we should have constant for it somewhere.

Reply:
Replaced with `INTERVAL` constant.